wetterdienst/dwd/radar/metadata.py

```diff
@@ -0,0 +1,111 @@
+from enum import Enum
+
+
+class RadarDate(Enum):
+
+    LATEST = "latest"
+    CURRENT = "current"
+    MOST_RECENT = "most_recent"
+
+
+class RadarDataFormat(Enum):
+    """
+    Enumeration for different radar data formats.
+    """
+
+    BINARY = "binary"
+    BUFR = "bufr"
+    HDF5 = "hdf5"
+
+
+class RadarDataSubset(Enum):
+    """
+    Enumerate HDF5 subset types for radar sweep data.
+
+    https://opendata.dwd.de/weather/radar/sites/sweep_pcp_v/boo/hdf5/
+    """
+
+    SIMPLE = "simple"
+    POLARIMETRIC = "polarimetric"
+
+
+class RadarParameter(Enum):
+
+    # Composites
+    # https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#German-Weather-Service:-RADOLAN-(quantitative)-composit  # noqa:E501,B950
+
+    # https://opendata.dwd.de/weather/radar/composit/
+    # TODO: "FX" was missing here. OTOH, "pp" doesn't
+    #       seem to be present on opendata.dwd.de.
+    FX_REFLECTIVITY = "fx"
+    PG_REFLECTIVITY = "pg"
+    # PP_REFLECTIVITY = "pp"
+    WX_REFLECTIVITY = "wx"
+    WN_REFLECTIVITY = "wn"
+    RX_REFLECTIVITY = "rx"
+
+    # https://opendata.dwd.de/weather/radar/radolan/
+    RW_REFLECTIVITY = "rw"
+    RY_REFLECTIVITY = "ry"
+    SF_REFLECTIVITY = "sf"
+
+    # Sites
+    # https://opendata.dwd.de/weather/radar/sites/
+    # https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#German-Weather-Service:-DX-format  # noqa:E501,B950
+    DX_REFLECTIVITY = "dx"
+    LMAX_VOLUME_SCAN = "lmax"
+    PE_ECHO_TOP = "pe"
+    PF_REFLECTIVITY = "pf"
+    PL_VOLUME_SCAN = "pl"
+    PR_VELOCITY = "pr"
+    PX_REFLECTIVITY = "px"
+    PX250_REFLECTIVITY = "px250"
+    PZ_CAPPI = "pz"
+
+    # OPERA HDF5 (ODIM_H5)
+    # https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#HDF5
+    SWEEP_PCP_VELOCITY_H = "sweep_pcp_v"
+    SWEEP_PCP_REFLECTIVITY_H = "sweep_pcp_z"
+    SWEEP_VOL_VELOCITY_H = "sweep_vol_v"
+    SWEEP_VOL_REFLECTIVITY_H = "sweep_vol_z"
+
+    # https://opendata.dwd.de/climate_environment/CDC/grids_germany/daily/radolan/
+    # https://opendata.dwd.de/climate_environment/CDC/grids_germany/hourly/radolan/
+    # https://opendata.dwd.de/climate_environment/CDC/grids_germany/5_minutes/radolan/
+    RADOLAN_CDC = "radolan_cdc"
+
+
+RADAR_PARAMETERS_COMPOSITES = [
+    # RadarParameter.PP_REFLECTIVITY,
```

Comment:
@meteoDaniel: You listed `PP_REFLECTIVITY` within `RADAR_PARAMETERS_COMPOSITES` the other day.